distribution/pom.xml

```diff
@@ -101,12 +101,13 @@
                         ${basedir}/../serialization/avro/src/main/java;
                         ${basedir}/../backends/jgroups/src/main/java;
                         ${basedir}/../backends/jms/src/main/java;
+                        ${basedir}/../elasticsearch/src/main/java;
                     </sourcepath>
                     <docfilessubdirs>true</docfilessubdirs>
                     <stylesheetfile>${basedir}/src/main/javadoc/stylesheet.css</stylesheetfile>
                     <links>
-                        <link>http://download.oracle.com/javase/7/docs/api/</link>
-                        <link>http://docs.jboss.org/hibernate/orm/5.0/javadocs/</link>
+                        <link>http://download.oracle.com/javase/8/docs/api/</link>
```

Comment:
Well spotted. Maybe we should also update the Lucene doc link to `5_4_1`? It seems there isn't any javadoc for later `5.x` versions, though...